### PR TITLE
M2-528 Builder - about applet page

### PR DESCRIPTION
**Pull request by wildkiomi**
image

### CODE REVIEW

File: src/components/FormComponents/InputController/InputController.tsx [hunk 1]
```diff
@@ -32,34 +38,43 @@ export const InputController = <T extends FieldValues>({
...
-                  : InputProps
-              }
-            />
+            <StyledTextFieldContainer>
+              <StyledTextField
+                {...textFieldProps}
+                onChange={onChange}
+                value={textFieldProps.type === 'number' && value < 1 ? 1 : value}
+                error={!!error || providedError}
+                helperText={error ? error.message : null}
```
Comment (kdanilenka32) [inline src/components/FormComponents/InputController/InputController.tsx:47]: error?.message || null

File: src/components/FormComponents/InputController/InputController.styles.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { styled, Box, TextField } from '@mui/material';
 
+import { StyledBodyMedium } from 'styles/styledComponents/Typography';
```
Comment (kdanilenka32) [inline src/components/FormComponents/InputController/InputController.styles.ts:3]: now you can import 'styles/styledComponents';

File: src/components/Uploader/Uploader.styles.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    width: ${width}rem;
+    border: ${variables.borderWidth.lg} ${isImgUploaded ? 'solid' : 'dashed'} ${
+    variables.palette.outline_variant
+  };
+`}
+`;
+
+export const StyledImgContainer = styled(Box)`
+  margin: 0 auto;
+  text-align: center;
```
Comment (kdanilenka32) [inline src/components/Uploader/Uploader.styles.ts:47]: space

File: src/components/Uploader/Uploader.styles.ts [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      border-left: transparent;
+    }
+    &:not(:last-of-type):hover {
+      border-right-color: transparent;
+    }
+  }
+
+  .MuiButton-root.MuiButton-outlined {
+    background-color: ${variables.palette.white};
+    border-color: ${variables.palette.surface_variant};
```
Comment (kdanilenka32) [inline src/components/Uploader/Uploader.styles.ts:75]: space

File: src/components/Uploader/Uploader.styles.ts [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  position: relative;
+`;
+
+export const StyledButtonGroup = styled(ButtonGroup)`
+  ${absolutePosition}
+
+  .MuiButtonGroup-grouped {
+    &:not(:first-of-type) {
+      border-left: transparent;
+    }
```
Comment (kdanilenka32) [inline src/components/Uploader/Uploader.styles.ts:67]: space

File: src/components/Uploader/Uploader.styles.ts [hunk 6]
```diff
@@ -0,0 +1,94 @@
+import { Box, ButtonGroup, FormControlLabel, TextField } from '@mui/material';
+import { styled } from '@mui/system';
+
+import { StyledFlexAllCenter } from 'styles/styledComponents/Flex';
```
Comment (kdanilenka32) [inline src/components/Uploader/Uploader.styles.ts:4]: 'styles/styledComponents"

File: src/components/Uploader/Uploader.tsx [hunk 7]
```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react';
+import { Box, Button } from '@mui/material';
+import { Trans } from 'react-i18next';
+
+import { StyledBodyLarge } from 'styles/styledComponents/Typography';
+import { Svg } from 'components/Svg';
```
Comment (kdanilenka32) [inline src/components/Uploader/Uploader.tsx:6]: 'components'

File: src/features/Builder/AboutApplet/AboutApplet.styles.ts [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  justify-content: flex-end;
+`;
+
+export const StyledSvg = styled(Svg)`
+  fill: ${variables.palette.outline};
+`;
+
+export const StyledTitle = styled(StyledTitleMedium)`
+  display: flex;
+  margin-bottom: ${theme.spacing(1.6)};
```
Comment (kdanilenka32) [inline src/features/Builder/AboutApplet/AboutApplet.styles.ts:39]: space

File: src/features/Builder/AboutApplet/AboutApplet.tsx [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+    mode: 'onChange',
+  });
+
+  const commonProps = {
+    control,
+    fullWidth: true,
+  };
+
+  return (
+    <StyledWrapper sx={{ marginRight: theme.spacing(20) }}>
```
Comment (kdanilenka32) [inline src/features/Builder/AboutApplet/AboutApplet.tsx:48]: you can use StyledBuilderWrapper

File: src/components/Uploader/Uploader.tsx [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+      if (e.dataTransfer.files[0]) {
+        setImageUrl(URL.createObjectURL(e.dataTransfer.files[0]));
+      }
+    },
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.files?.[0]) {
+      setImageUrl(URL.createObjectURL(event.target.files?.[0]));
+    }
```
Comment (kdanilenka32) [inline src/components/Uploader/Uploader.tsx:44]: const { files } = event.target
if (files?.[0]) {
      setImageUrl(URL.createObjectURL(files.[0]));
    }

File: src/features/Builder/AboutApplet/AboutApplet.tsx [hunk 11]
```diff
@@ -0,0 +1,135 @@
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { EditorController, InputController, SelectController } from 'components/FormComponents';
+import { StyledFlexTopCenter } from 'styles/styledComponents/Flex';
```
Comment (kdanilenka32) [inline src/features/Builder/AboutApplet/AboutApplet.tsx:6]: styles/styledComponents

File: src/features/Builder/AboutApplet/AboutApplet.tsx [hunk 12]
```diff
@@ -0,0 +1,135 @@
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { EditorController, InputController, SelectController } from 'components/FormComponents';
+import { StyledFlexTopCenter } from 'styles/styledComponents';
+import { useBreadcrumbs } from 'hooks';
+import { Svg, Tooltip, Uploader } from 'components';
+import { StyledBodyMedium, StyledHeadlineLarge } from 'styles/styledComponents';
```
Comment (IhorNesterenko99) [inline src/features/Builder/AboutApplet/AboutApplet.tsx:9]: pls merge 6, 9, 10 rows imports from styledComponents into one import

File: src/pages/Builder/Builder.tsx [hunk 13]
```diff
@@ -3,17 +3,20 @@ import { useNavigate } from 'react-router-dom';
 
 import { useBreadcrumbs } from 'hooks';
 import { page } from 'resources';
+import { StyledBody } from 'styles/styledComponents/Body';
 import { StyledFlexAllCenter } from 'styles/styledComponents';
```
Comment (IhorNesterenko99) [inline src/pages/Builder/Builder.tsx:7]: pls merge imports from styled components into one

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by kdanilenka32 — APPROVED** [review]

**Pull request merged**